public/index.html

```diff
@@ -7,15 +7,19 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="E-commerce app for quality products"
+      content="Web site created using create-react-app"
```

Comment:
Hello @GbemiMajek, please pull again and accept incoming changes, not current changes, for this file. I will send a screenshot of the diff to you